src/modules/flow/utils/domParserUtils.ts

```
@@ -0,0 +1,44 @@
+
+// Check if we're in a browser environment
+const isBrowser =
+  typeof window !== "undefined" && typeof window.DOMParser !== "undefined";
+
+// DOMParser implementation for Node.js
+let DOMParserImpl: any;
+
+if (!isBrowser) {
+  // In Node.js, use the xmldom implementation
+  try {
+    const { DOMParser } = require("@xmldom/xmldom");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)